docs/sources/mimir/configure/configure-object-storage-backend.md

```diff
@@ -83,6 +83,10 @@ ruler_storage:
     bucket_name: mimir-ruler
 ```
 
+{{< admonition type="note" >}}
+The endpoint value must not be prefixed with your bucket name (e.g., if your bucket is "my-storage-bucket", the endpoint cannot be "my-storage-bucket.s3.amazonaws.com").
```

Comment:
```suggestion
Don't prefix the endpoint value with your bucket name. For example, if your bucket is named "my-storage-bucket", you cannot name the endpoint "my-storage-bucket.s3.amazonaws.com").
```

Reply:
@tacole02 I've addressed the comment, please have a look